.gitignore

```diff
@@ -1,2 +1,3 @@
 firectl
 firectl.sha256
+.vscode
```

Comment:
For editor-specific files, I think it would be better to use a per-user global gitignore file.

https://help.github.com/en/github/using-git/ignoring-files#configuring-ignored-files-for-all-repositories-on-your-computer